### DISCUSSION TITLE
Requirements confllict between astronomer-providers 1.4  and constraints file for Astro 2.3.2

### DISCUSSION THREAD

**Discussion by neelavarrao** [astronomer/astronomer-providers#440] [Q&A]
I see that the `google-api-core` version requirements creates a conflict between the astronomer-providers and the astro 2.3.2 constraints file and that has been removed in 1.5. Just reaching out to see if there is a timeline for 1.5 release?

**Comment by neelavarrao** [comment]
https://github.com/astronomer/astronomer-providers/blob/1.4.0/setup.cfg#L67

https://github.com/astronomer/ap-airflow/blob/master/2.3.2/bullseye/build-time-pip-constraints.txt#L241

**Comment by bharanidharan14** [comment]
@neelavarrao  There is a latest release  `astronomer-providers 1.7.0`,  can you please check and if you find any issue let us know

**Comment by pankajkoti** [comment]
Closing this discussion as it appears stale. Please comment if needs to be reopened.
